File: Snooper/src/main/java/com/prateekj/snooper/networksnooper/views/HttpCallBodyView.java
package com.prateekj.snooper.networksnooper.views;


import com.prateekj.snooper.networksnooper.model.Bound;

import java.util.List;

public interface HttpCallBodyView {
  void onFormattingDone();
  void highlightBounds(List<Bound> bounds);
  void removeOldHighlightedSpans();
}

File: Snooper/src/main/java/com/prateekj/snooper/networksnooper/views/HttpCallBodyView.kt
package com.prateekj.snooper.networksnooper.views


import com.prateekj.snooper.networksnooper.model.Bound

interface HttpCallBodyView {
  fun onFormattingDone()
  fun highlightBounds(bounds: List<Bound>)
  fun removeOldHighlightedSpans()
}
